lib/Doctrine/Common/Cache/Psr6/DoctrineProvider.php

```diff
@@ -122,4 +123,12 @@ protected function doGetStats()
     {
         return null;
     }
+
+    public function reset()
```

Comment:
We do not require an interface for this as the `service_resetter` of symfony doesn't require anything here: https://github.com/symfony/symfony/blob/6.1/src/Symfony/Component/HttpKernel/DependencyInjection/ServicesResetter.php